packages/apfelxx/install.sh

```diff
@@ -0,0 +1,26 @@
+# early exit if errors occur
+set -e
+
+INSTALLER_DIR=$(realpath $(dirname $0))
+
+# import variables
+. $INSTALLER_DIR/variables.sh
+
+# download the code
+sh $INSTALLER_DIR/download.sh $URL
+
+# extract code
+cd $SRC_DIR
+rm -rf build
+mkdir build && cd build
+
+# if PREFIX is set use it, otherwise do the default
+if [ -z $PREFIX ]; then
+  cmake .. 
+else
+  echo "Installing in $PREFIX"
+  cmake -DCMAKE_INSTALL_PREFIX=$PREFIX ..
+fi
+
+# compile, and install (includes python interface)
+make && make install
```

Comment:
Keep new lines at the end of files, it is a good practice (you can look up on the web for the reason).

You can set your editor to do it, and their absence on GitHub is signaled by this symbol:
<img width="75" alt="image" src="https://user-images.githubusercontent.com/10830876/225253814-440383f1-e30b-419d-8807-aec8b34689ea.png">